### ISSUE TITLE
CORS and Cloudeflare problems

### ISSUE THREAD

**Issue by JuergenGutsch** [utterance/utterances#91]
Hi @jdanyow 

I just changed may blog comments from Disqus to Utterances and got several problems:

1. CORS: as a logged-on user while trying to post a comment:
![image](https://user-images.githubusercontent.com/365688/48698367-2d9c7080-ebe7-11e8-864d-aa3efba1c960.png)

2. Cloudeflare while trying to logon:
![image](https://user-images.githubusercontent.com/365688/48698483-7bb17400-ebe7-11e8-89f4-b44241be0250.png)
https://twitter.com/davepermen/status/1064450165057167360

Is this a new problem? Didn't get this errors last week while trying it out.
Did I forget something?

Here is my blog with the new Utterances comments:
https://asp.net-hacker.rocks/

EDIT:
The CORS error happens on this site as well:
https://www.swifttalking.com/writing-mocks-manually-in-swift/

Thanks,
Juergen

**Comment by jdanyow** [comment]
Thanks for reporting this!

Here's what happened...

1. I use Cloudflare DNS to route `//utteranc.es` requests to the static website hosted on GitHub pages and to route `//api.utteranc.es` requests to the node app hosted on Azure.
1. About a week ago I created a Cloudflare worker that took over for the node app hosted on Azure. The worker is configured to responds to all requests to `//api.utteranc.es`. Once I turned this on, no requests ever made it to the node app on Azure. As a precaution, I left the node app on Azure running, in case something went wrong with the Cloudflare worker and I needed to switch back.
1. Fast-forward to yesterday afternoon... I said to myself *the Cloudflare worker is doing very well, let's delete the Azure app to save a little :moneybag:*. Here's where things went wrong... moments after I deleted the Azure app, I tested the API, it was up so I thought everything was good. I had forgotten I still had a DNS entry in Cloudflare to route `//api.utteranc.es` requests to Azure. Once Cloudflare noticed the Azure app was down, it stopped running the Cloudflare worker on the `//api.utteranc.es` requests which meant the utterance API was down.
1. Thankfully @JuergenGutsch reported this issue here and [on twitter](https://twitter.com/sharpcms/status/1064451016308006912).
1. To fix the issue I edited the DNS entry for `api.utteranc.es` that was pointing at Azure, switching it to point at github pages (it just needs to be resolvable, Cloudflare workers handle all the requests to `api.utteranc.es`).
